### DISCUSSION TITLE
Migrate from other CMS to Statamic

### DISCUSSION THREAD

**Discussion by richardhj** [statamic/cms#8080] [Q&A]
Hello!

I am wondering if there is an easy way to migrate from other CMSes or arbitrary sources to Statamic and Bard.

In particular, I am asking if there is an optimal way to programmatically create Bard content like:

```md
content:
  -
    type: paragraph
    content:
      -
        type: text
        text: 'Lorem ipsum'
---
```

If so, I could easily write a migration script for my app.

Thanks!

**Comment by jasonvarga** [comment]
If you put html in the markdown file, Bard should know how to handle it. (As long as there's no overcomplicated html in there)

```yaml
content: '<p>Lorem ipsum</p>`
```

You will need to make sure that you've enabled the buttons for the HTML elements you plan to use.

For example, if your html has `<b>` tags in it, the bold button will need to be enabled in order for it to render.
